src/modules/getGameID.js

```diff
@@ -0,0 +1,11 @@
+const getGameID = () => {
+  const savedGameID = localStorage.getItem('gameID');
+
+  if (savedGameID !== null) {
+    const gameID = JSON.parse(savedGameID);
+    return gameID;
+  }
+  return [];
+};
```

Comment:
- [ ] I wanted to highlight a challenge we're facing right after launching the app. Currently, we're encountering an issue where we're unable to make `POST` or `GET` requests immediately after the app starts. This problem seems to stem from the fact that the `GameID` is empty in the local storage as soon as the app is launched, which necessitates a page reload to obtain the necessary `GameID`. To enhance user experience and avoid this initial hurdle, I suggest exploring a solution that allows us to obtain the `GameID` seamlessly without the need for a page reload. Resolving this matter would greatly contribute to the smooth functioning of our application right from the start.